### PR TITLE
[EventHubs & ServiceBus] Async sample should use asyncio.run

### PR DESCRIPTION
**Pull request by yunhaoling**

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/samples/async_samples/sample_code_eventhub_async.py [hunk 1]
```diff
@@ -207,8 +207,6 @@ async def on_event(partition_context, event):
     # [END eventhub_consumer_client_close_async]
 
 if __name__ == '__main__':
-    loop = asyncio.get_event_loop()
-    # loop.run_until_complete(example_eventhub_async_consumer_receive_and_close())
-    # loop.run_until_complete(example_eventhub_async_producer_send_and_close())
-    loop.run_until_complete(example_eventhub_async_send_and_receive())
-
+    # asyncio.run(example_eventhub_async_consumer_receive_and_close())
+    # asyncio.run(example_eventhub_async_producer_send_and_close())
```
Comment (swathipil) [inline sdk/eventhub/azure-eventhub/samples/async_samples/sample_code_eventhub_async.py:211]: nit:
These may need to be uncommented? I think I may have been testing and accidentally committed them like this one time 🤦 this one's on me!

File: sdk/servicebus/azure-servicebus/samples/async_samples/auto_lock_renew_async.py [hunk 2]
```diff
@@ -127,8 +127,6 @@ async def on_lock_renew_failure_callback(renewable, error):
...
 
-
-loop = asyncio.get_event_loop()
-loop.run_until_complete(renew_lock_on_message_received_from_non_sessionful_entity())
-loop.run_until_complete(renew_lock_on_session_of_the_sessionful_entity())
-loop.run_until_complete(renew_lock_with_lock_renewal_failure_callback())
\ No newline at end of file
+asyncio.run(renew_lock_on_message_received_from_non_sessionful_entity())
+asyncio.run(renew_lock_on_session_of_the_sessionful_entity())
+asyncio.run(renew_lock_with_lock_renewal_failure_callback())
```
Comment (swathipil) [inline sdk/servicebus/azure-servicebus/samples/async_samples/auto_lock_renew_async.py:132]: baby nit:
newline might be missing?

**Review by swathipil — APPROVED** [review]
a couple of small nits, but o/w everything looks good to me!

File: sdk/servicebus/azure-servicebus/samples/async_samples/auto_lock_renew_async.py (hunk 2 above)
Comment (yunhaoling) [reply sdk/servicebus/azure-servicebus/samples/async_samples/auto_lock_renew_async.py]: updated

File: sdk/eventhub/azure-eventhub/samples/async_samples/sample_code_eventhub_async.py (hunk 1 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/samples/async_samples/sample_code_eventhub_async.py]: good catch, uncommented

**Pull request merged**